source/infrastructure/hosting/azure-cip/index.html.md.erb

```diff
@@ -34,6 +34,34 @@ The new user will receive an invitation by email. Then the service managers can
 
 To access Azure Devops, the new user must access [Azure DevOps](https://dev.azure.com/dfe-ssp/). This will register them in the system then the service owner can add them to the project.
 
+## Privileged Identity Management (PIM) Requests
+
+Privileged Identity Management (PIM) is a service in Azure Active Directory (Azure AD) that enables you to manage, control, and monitor access to important resources at DfE such as access to staging and production environments.
+
+To request access to your eligible assignments, follow the steps below:
+
+* Navigate to the [Azure Portal](https://portal.azure.com/) and login using your `@digital.education.gov.uk` account
+
+* Click on `Directories + subscriptions`, this is the icon to the left of the notification bell icon in the blue navigation bar
+
+* Ensure you are in the `DfE Platform Identity` directory, if you are not, switch to it. You may be prompted to verify your identity with 2FA
+
+* Search for `PIM` in the search bar at the top of the page and select `Azure AD privileged Identity Management`
+
+* Click on `My roles` on the left hand side of the page, under `Tasks`
+
+* Click on `Azure resources` under `Activate`
+
+* You may have to lengthen the resource section in order to see the full resource name, including the environment
+
+* Click `Activate` under `Action` for the relevant resource
+
+* Select a duration (maximum 8 hours), enter a reason for the request and click `Activate`
+
+* An email will be sent to the administrators who can approve or deny the request
```

Comment:
For the test subscription, approval is automatic and you should receive confirmation shortly.
For the production subscription, an email will be sent...

Reply:
Thanks, updated this. I think the test subscription is the staging environment, is this always usually the case?

Reply:
This is only the case for Teacher services, so I think we should be more generic. Maybe:
>The test subscription (which usually includes staging, preprod, etc. environments), is self approving...

Reply:
Have also updated